packages/TorneloScoresheet/src/pages/ResultDisplay/ResultDisplay.tsx

```diff
@@ -21,8 +22,25 @@ const ResultDisplay: React.FC = () => {
   }`;
 
   const handleEmailGame = (): void => {
-    // TODO: Implement email logic
-    console.log('email game button pressed');
+    const subject =
+      'Tornello Game Results ' +
+      infoString +
+      ':' +
+      resultDisplayMode?.pairing.players[0].firstName +
+      ' vs' +
+      resultDisplayMode?.pairing.players[1].firstName;
+    const body =
+      'Tornello Game Results ' +
+      infoString +
+      '\n\nPlayer 1: ' +
+      resultDisplayMode?.pairing.players[0].firstName +
+      resultDisplayMode?.pairing.players[0].lastName +
```

Comment:
We might not need to include the player names here since it's already in both the subject of the email and the pgn. But if you want to include it we have a function in player  (in utils) called fullName which returns the full name of the player so we should probably use that instead to shorten the code.

Reply:
I've changed it to use the fullName function - I think I'll keep the names in the email body since they are a bit harder to find the names in the PGN.